### PR TITLE
Add label for IPCEI projects

### PR DESCRIPTION
**Pull request by oliver-goetz**
/kind enhancement

**What this PR does / why we need it**:
This PR adds a label to label issues and PRs related to IPCEI projects.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: config/prow/labels.yaml [hunk 1]
```diff
@@ -356,6 +356,12 @@ repos:
         addedBy: anyone
         previously:
           - name: discussion
+      - name: kind/ipsei
```
Comment (rfranzke) [inline config/prow/labels.yaml:359]: typo?

Also, why `kind/` prefix? This doesn't make much sense to me. We use `kind/bug`, `kind/test`, `kind/flake`, `kind/enhancement`, etc., but `kind/ipcei` doesn't seem to fit into this category.

Comment (oliver-goetz) [reply config/prow/labels.yaml:359]: Apparently, it is a typo. I just copied it from @vlerenc's gardener-robot config and did not question it. Funny 😅 

I agree, `kind/ipcei` does not fit to the other members of the category. Maybe `area/ipcei`? This looks similiar to `area/open-source`. WDYT @vlerenc @rfranzke?

Comment (vlerenc) [reply config/prow/labels.yaml]: Yes, @dkistner saw that, too. I fixed it in the bot now.

As for `kind`, `area`, nothing, `granny`, `mars`, I am very open. Maybe someone else takes the decision and tells me whether I should rename it?

**Review by timebertt — COMMENTED** [review]
/lgtm

**Review by timuthy — COMMENTED** [review]
/lgtm

File: config/prow/labels.yaml (hunk 1 above)
Comment (oliver-goetz) [reply config/prow/labels.yaml]: afaik we aligned on `area/ipcei` now

**Pull request merged**